templates/test/ci/cluster-template-prow-ci-version-windows-containerd-2022.yaml

```diff
@@ -1,1179 +0,0 @@
-apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
this template is not tracked by any test flavor and should have been removed as part of https://github.com/kubernetes-sigs/cluster-api-provider-azure/commit/8929290de56ad53e62d59bf0842db7baa4e15b72, it's already removed from the main branch